frontend/views/map.js

```diff
@@ -88,7 +89,19 @@ module.exports = Backbone.View.extend({
         }
         $('.no-search-results').hide();
         $('#map_container').show();
-        $('.results-truncated').toggle(search.has_more);
+
+        if (services.length < search.total_results) {
+            msg = i18n.t("Service-Map.More-Results", {
+                // ARGH!  "count" is a magic var name to i18n and won't work here,
```

Comment:
On Wed, Jun 24, 2015 at 4:29 PM, Mark Lavin <notifications@github.com>
wrote:

> Just so I'm understanding this comment correctly, this is meant to explain
> why the variable is named thecount rather than simply count right?
>

​Right.  I've clarified the comment.